public/css/dashboard.css

```diff
@@ -488,7 +500,7 @@ body {
                 var(--main-background-color) 100%)
     ;
         grid-template-rows: auto auto 1fr;
-        grid-template-columns: auto 1fr 1fr 1fr;
+        grid-template-columns: auto 1fr 1fr 200px;
```

Comment:
Why is the last column fixed width? We do not have a column on the right in our layout as far as I'm aware?

Reply:
Ok, I see what you mean. I changed the width of the user menu instead